src/guides/compile-and-run.md

```diff
@@ -31,14 +31,14 @@ greeting.
 
 ```
 🏁 🍇
-  😀 🔤Hey!🔤❗️
+  😀 🔤👋🌍🔤❗️
 🍉
 ```
 
 Before analyzing this new code, we’ll give it a try.
 
->!H You must have installed Emojicode to run the following commands.
->!H See [Installing Emojicode](install.html) if you haven’t already.
+> !H You must have installed Emojicode to run the following commands.
+> !H See [Installing Emojicode](install.html) if you haven’t already.
```

Comment:
Theses spaces break the callout: https://deploy-preview-89--emojicode.netlify.app/docs/guides/compile-and-run.html#greetings